### ISSUE TITLE
Alphafold not running with multimer mode and Output per-residue confidence scores

### ISSUE THREAD

**Issue by tchaussepiedifb** [usegalaxy-au/tools-au#44]
Hi,
We got an error report where Alphafold with `multimer mode` and `Output per-residue confidence scores` cannot generated residue_scores because of `KeyError: 'model_1'`.
Are the multimer mode and Output per-residue confidence scores compatible ?

```
I0927 11:48:46.404397 47533421533248 templates.py:857] Using precomputed obsolete pdbs /shared/bank/alphafold2/current/pdb_mmcif/obsolete.dat.
I0927 11:48:47.067361 47533421533248 tpu_client.py:50] Starting the local TPU driver.
I0927 11:48:47.092056 47533421533248 xla_bridge.py:212] Unable to initialize backend 'tpu_driver': NOT_FOUND: Unable to find driver in registry given worker: local://
I0927 11:48:48.270602 47533421533248 xla_bridge.py:212] Unable to initialize backend 'tpu': INVALID_ARGUMENT: TpuPlatform is not available.
I0927 11:48:52.628931 47533421533248 run_alphafold.py:398] Have 5 models: ['model_1_multimer', 'model_2_multimer', 'model_3_multimer', 'model_4_multimer', 'model_5_multimer']
I0927 11:48:52.629949 47533421533248 run_alphafold.py:414] Using random seed 519914793107545959 for the data pipeline
I0927 11:48:52.631131 47533421533248 run_alphafold.py:163] Predicting alphafold
I0927 11:48:52.638058 47533421533248 pipeline_multimer.py:211] Running monomer pipeline on chain A: 6Y4F_1|Chain A|Fimbrial adhesin|Proteus mirabilis (strain HI4320) (529507)
I0927 11:48:52.639432 47533421533248 jackhmmer.py:133] Launching subprocess "/usr/bin/jackhmmer -o /dev/null -A /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmpypkh7122/output.sto --noali --F1 0.0005 --F2 5e-05 --F3 5e-07 --incE 0.0001 -E 0.0001 --cpu 8 -N 1 /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmp40_uf3re.fasta /shared/bank/alphafold2/current/uniref90/uniref90.fasta"
I0927 11:48:52.705271 47533421533248 utils.py:36] Started Jackhmmer (uniref90.fasta) query
I0927 11:54:02.665200 47533421533248 utils.py:40] Finished Jackhmmer (uniref90.fasta) query in 309.960 seconds
I0927 11:54:02.675121 47533421533248 jackhmmer.py:133] Launching subprocess "/usr/bin/jackhmmer -o /dev/null -A /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmp18ds6wrr/output.sto --noali --F1 0.0005 --F2 5e-05 --F3 5e-07 --incE 0.0001 -E 0.0001 --cpu 8 -N 1 /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmp40_uf3re.fasta /shared/bank/alphafold2/current/mgnify/mgy_clusters_2018_12.fa"
I0927 11:54:02.689186 47533421533248 utils.py:36] Started Jackhmmer (mgy_clusters_2018_12.fa) query
I0927 11:59:25.699578 47533421533248 utils.py:40] Finished Jackhmmer (mgy_clusters_2018_12.fa) query in 323.010 seconds
I0927 11:59:25.710572 47533421533248 hmmbuild.py:121] Launching subprocess ['/usr/bin/hmmbuild', '--hand', '--amino', '/shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmpmvvwxt4w/output.hmm', '/shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmpmvvwxt4w/query.msa']
I0927 11:59:25.854263 47533421533248 utils.py:36] Started hmmbuild query
I0927 11:59:25.968475 47533421533248 hmmbuild.py:129] hmmbuild stdout:
# hmmbuild :: profile HMM construction from multiple sequence alignments
# HMMER 3.1b2 (February 2015); http://hmmer.org/
# Copyright (C) 2015 Howard Hughes Medical Institute.
# Freely distributed under the GNU General Public License (GPLv3).
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# input alignment file:             /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmpmvvwxt4w/query.msa
# output HMM file:                  /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmpmvvwxt4w/output.hmm
# input alignment is asserted as:   protein
# model architecture construction:  hand-specified by RF annotation
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

# idx name                  nseq  alen  mlen eff_nseq re/pos description
#---- -------------------- ----- ----- ----- -------- ------ -----------
1     query                  125   385   141     2.35  0.590 

# CPU time: 0.10u 0.01s 00:00:00.11 Elapsed: 00:00:00.11


stderr:


I0927 11:59:25.968567 47533421533248 utils.py:40] Finished hmmbuild query in 0.114 seconds
I0927 11:59:25.975533 47533421533248 hmmsearch.py:103] Launching sub-process ['/usr/bin/hmmsearch', '--noali', '--cpu', '8', '--F1', '0.1', '--F2', '0.1', '--F3', '0.1', '--incE', '100', '-E', '100', '--domE', '100', '--incdomE', '100', '-A', '/shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmpqj8qr4he/output.sto', '/shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmpqj8qr4he/query.hmm', '/shared/bank/alphafold2/current/pdb_seqres/pdb_seqres.txt']
I0927 11:59:25.989840 47533421533248 utils.py:36] Started hmmsearch (pdb_seqres.txt) query
I0927 11:59:28.916383 47533421533248 utils.py:40] Finished hmmsearch (pdb_seqres.txt) query in 2.926 seconds
I0927 11:59:28.928347 47533421533248 jackhmmer.py:133] Launching subprocess "/usr/bin/jackhmmer -o /dev/null -A /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmp7rjs600j/output.sto --noali --F1 0.0005 --F2 5e-05 --F3 5e-07 --incE 0.0001 -E 0.0001 --cpu 8 -N 1 /shared/ifbstor1/galaxy/jobs/002/024/2024327/tmp/tmp40_uf3re.fasta /shared/bank/alphafold2/current/small_bfd/bfd-first_non_consensus_sequences.fasta"
I0927 11:59:28.947222 47533421533248 utils.py:36] Started Jackhmmer (bfd-first_non_consensus_sequences.fasta) query
I0927 12:01:01.773897 47533421533248 utils.py:40] Finished Jackhmmer (bfd-first_non_consensus_sequences.fasta) query in 92.826 seconds
I0927 12:01:01.778645 47533421533248 templates.py:940] Searching for template for: SIFSYITESTGTPSNATYTYVIERWDPETSGILNPCYGWPVCYVTVNHKHTVNGTGGNPAFQIARIEKLRTLAEVRDVVLKNRSFPIEGQTTHRGPSLNSNQECVGLFYQPNSSGISPRGKLLPGSLCGIAPPPVHHHHHH
I0927 12:01:01.779139 47533421533248 templates.py:718] hit 6y4f_A did not pass prefilter: Template is an exact subsequence of query with large coverage. Length ratio: 0.9929078014184397.
I0927 12:01:02.094127 47533421533248 templates.py:268] Found an exact template match 6y4e_A.
I0927 12:01:02.202072 47533421533248 templates.py:268] Found an exact template match 6kzr_A.
I0927 12:01:02.206404 47533421533248 templates.py:268] Found an exact template match 6kzr_B.
I0927 12:01:02.361718 47533421533248 templates.py:268] Found an exact template match 6lfj_A.
I0927 12:01:02.366056 47533421533248 templates.py:268] Found an exact template match 6lfj_B.
I0927 12:01:02.694677 47533421533248 templates.py:268] Found an exact template match 6lkr_A.
I0927 12:01:02.698958 47533421533248 templates.py:268] Found an exact template match 6lkr_B.
I0927 12:01:02.788154 47533421533248 templates.py:268] Found an exact template match 1txu_A.
I0927 12:01:55.850371 47533421533248 templates.py:268] Found an exact template match 6u42_7C.
I0927 12:01:56.027099 47533421533248 templates.py:268] Found an exact template match 1jw1_A.
I0927 12:01:56.046079 47533421533248 templates.py:268] Found an exact template match 1jw1_A.
I0927 12:01:56.135943 47533421533248 templates.py:268] Found an exact template match 3rkv_A.
I0927 12:01:56.429510 47533421533248 templates.py:268] Found an exact template match 1pc3_A.
I0927 12:01:56.439286 47533421533248 templates.py:268] Found an exact template match 1pc3_A.
I0927 12:01:56.448171 47533421533248 templates.py:268] Found an exact template match 1pc3_B.
I0927 12:01:56.457408 47533421533248 templates.py:268] Found an exact template match 1pc3_B.
I0927 12:01:57.051022 47533421533248 templates.py:268] Found an exact template match 7dm1_A.
I0927 12:01:57.061417 47533421533248 templates.py:268] Found an exact template match 7dm1_A.
I0927 12:01:57.070404 47533421533248 templates.py:268] Found an exact template match 7dm1_B.
I0927 12:01:57.080437 47533421533248 templates.py:268] Found an exact template match 7dm1_B.
I0927 12:01:57.384104 47533421533248 templates.py:268] Found an exact template match 7dm2_A.
I0927 12:01:57.393676 47533421533248 templates.py:268] Found an exact template match 7dm2_A.
I0927 12:01:57.470570 47533421533248 templates.py:268] Found an exact template match 2wfo_A.
I0927 12:01:57.581230 47533421533248 templates.py:268] Found an exact template match 5d1m_B.
I0927 12:01:57.589017 47533421533248 pipeline.py:239] Uniref90 MSA size: 125 sequences.
I0927 12:01:57.589076 47533421533248 pipeline.py:240] BFD MSA size: 6 sequences.
I0927 12:01:57.589116 47533421533248 pipeline.py:241] MGnify MSA size: 24 sequences.
I0927 12:01:57.589146 47533421533248 pipeline.py:243] Final (deduplicated) MSA size: 143 sequences.
I0927 12:01:57.589426 47533421533248 pipeline.py:246] Total number of templates (NB: this can include bad templates and is later filtered to top 4): 11.
I0927 12:01:57.685484 47533421533248 run_alphafold.py:198] Running model model_1_multimer on alphafold
I0927 12:01:57.686075 47533421533248 model.py:166] Running predict with shape(feat) = {'aatype': (564,), 'residue_index': (564,), 'seq_length': (), 'msa': (512, 564), 'num_alignments': (), 'template_aatype': (4, 564), 'template_all_atom_mask': (4, 564, 37), 'template_all_atom_positions': (4, 564, 37, 3), 'asym_id': (564,), 'sym_id': (564,), 'entity_id': (564,), 'deletion_matrix': (512, 564), 'deletion_mean': (564,), 'all_atom_mask': (564, 37), 'all_atom_positions': (564, 37, 3), 'assembly_num_chains': (), 'entity_mask': (564,), 'num_templates': (), 'cluster_bias_mask': (512,), 'bert_mask': (512, 564), 'seq_mask': (564,), 'msa_mask': (512, 564)}
I0927 12:10:00.069120 47533421533248 model.py:176] Output shape was {'distogram': {'bin_edges': (63,), 'logits': (564, 564, 64)}, 'experimentally_resolved': {'logits': (564, 37)}, 'masked_msa': {'logits': (252, 564, 22)}, 'predicted_aligned_error': (564, 564), 'predicted_lddt': {'logits': (564, 50)}, 'structure_module': {'final_atom_mask': (564, 37), 'final_atom_positions': (564, 37, 3)}, 'plddt': (564,), 'aligned_confidence_probs': (564, 564, 64), 'max_predicted_aligned_error': (), 'ptm': (), 'iptm': (), 'ranking_confidence': ()}
I0927 12:10:00.070375 47533421533248 run_alphafold.py:212] Total JAX model model_1_multimer on alphafold predict time (includes compilation time, see --benchmark): 482.4s
I0927 12:10:13.439882 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:10:13.874265 47533421533248 amber_minimize.py:408] Minimizing protein, attempt 1 of 100.
I0927 12:10:20.581728 47533421533248 amber_minimize.py:69] Restraining 4388 / 8616 particles.
I0927 12:10:32.419306 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:10:38.066517 47533421533248 amber_minimize.py:500] Iteration completed: Einit 23343.82 Efinal -11893.04 Time 8.79 s num residue violations 0 num residue exclusions 0 
I0927 12:10:46.333320 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:10:50.362350 47533421533248 run_alphafold.py:198] Running model model_2_multimer on alphafold
I0927 12:10:50.362713 47533421533248 model.py:166] Running predict with shape(feat) = {'aatype': (564,), 'residue_index': (564,), 'seq_length': (), 'msa': (512, 564), 'num_alignments': (), 'template_aatype': (4, 564), 'template_all_atom_mask': (4, 564, 37), 'template_all_atom_positions': (4, 564, 37, 3), 'asym_id': (564,), 'sym_id': (564,), 'entity_id': (564,), 'deletion_matrix': (512, 564), 'deletion_mean': (564,), 'all_atom_mask': (564, 37), 'all_atom_positions': (564, 37, 3), 'assembly_num_chains': (), 'entity_mask': (564,), 'num_templates': (), 'cluster_bias_mask': (512,), 'bert_mask': (512, 564), 'seq_mask': (564,), 'msa_mask': (512, 564)}
I0927 12:17:35.662647 47533421533248 model.py:176] Output shape was {'distogram': {'bin_edges': (63,), 'logits': (564, 564, 64)}, 'experimentally_resolved': {'logits': (564, 37)}, 'masked_msa': {'logits': (252, 564, 22)}, 'predicted_aligned_error': (564, 564), 'predicted_lddt': {'logits': (564, 50)}, 'structure_module': {'final_atom_mask': (564, 37), 'final_atom_positions': (564, 37, 3)}, 'plddt': (564,), 'aligned_confidence_probs': (564, 564, 64), 'max_predicted_aligned_error': (), 'ptm': (), 'iptm': (), 'ranking_confidence': ()}
I0927 12:17:35.663999 47533421533248 run_alphafold.py:212] Total JAX model model_2_multimer on alphafold predict time (includes compilation time, see --benchmark): 405.3s
I0927 12:17:45.817920 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:17:46.238741 47533421533248 amber_minimize.py:408] Minimizing protein, attempt 1 of 100.
I0927 12:17:46.777984 47533421533248 amber_minimize.py:69] Restraining 4388 / 8616 particles.
I0927 12:17:57.375577 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:17:58.749400 47533421533248 amber_minimize.py:500] Iteration completed: Einit 14410.28 Efinal -12176.97 Time 1.37 s num residue violations 0 num residue exclusions 0 
I0927 12:18:07.222037 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:18:08.096813 47533421533248 run_alphafold.py:198] Running model model_3_multimer on alphafold
I0927 12:18:08.097224 47533421533248 model.py:166] Running predict with shape(feat) = {'aatype': (564,), 'residue_index': (564,), 'seq_length': (), 'msa': (512, 564), 'num_alignments': (), 'template_aatype': (4, 564), 'template_all_atom_mask': (4, 564, 37), 'template_all_atom_positions': (4, 564, 37, 3), 'asym_id': (564,), 'sym_id': (564,), 'entity_id': (564,), 'deletion_matrix': (512, 564), 'deletion_mean': (564,), 'all_atom_mask': (564, 37), 'all_atom_positions': (564, 37, 3), 'assembly_num_chains': (), 'entity_mask': (564,), 'num_templates': (), 'cluster_bias_mask': (512,), 'bert_mask': (512, 564), 'seq_mask': (564,), 'msa_mask': (512, 564)}
I0927 12:25:08.053479 47533421533248 model.py:176] Output shape was {'distogram': {'bin_edges': (63,), 'logits': (564, 564, 64)}, 'experimentally_resolved': {'logits': (564, 37)}, 'masked_msa': {'logits': (252, 564, 22)}, 'predicted_aligned_error': (564, 564), 'predicted_lddt': {'logits': (564, 50)}, 'structure_module': {'final_atom_mask': (564, 37), 'final_atom_positions': (564, 37, 3)}, 'plddt': (564,), 'aligned_confidence_probs': (564, 564, 64), 'max_predicted_aligned_error': (), 'ptm': (), 'iptm': (), 'ranking_confidence': ()}
I0927 12:25:08.055119 47533421533248 run_alphafold.py:212] Total JAX model model_3_multimer on alphafold predict time (includes compilation time, see --benchmark): 420.0s
I0927 12:25:17.094351 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:25:17.515309 47533421533248 amber_minimize.py:408] Minimizing protein, attempt 1 of 100.
I0927 12:25:18.057983 47533421533248 amber_minimize.py:69] Restraining 4388 / 8616 particles.
I0927 12:25:25.816612 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:25:27.120573 47533421533248 amber_minimize.py:500] Iteration completed: Einit 67757.60 Efinal -12377.07 Time 1.99 s num residue violations 0 num residue exclusions 0 
I0927 12:25:41.352189 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:25:42.184409 47533421533248 run_alphafold.py:198] Running model model_4_multimer on alphafold
I0927 12:25:42.184644 47533421533248 model.py:166] Running predict with shape(feat) = {'aatype': (564,), 'residue_index': (564,), 'seq_length': (), 'msa': (512, 564), 'num_alignments': (), 'template_aatype': (4, 564), 'template_all_atom_mask': (4, 564, 37), 'template_all_atom_positions': (4, 564, 37, 3), 'asym_id': (564,), 'sym_id': (564,), 'entity_id': (564,), 'deletion_matrix': (512, 564), 'deletion_mean': (564,), 'all_atom_mask': (564, 37), 'all_atom_positions': (564, 37, 3), 'assembly_num_chains': (), 'entity_mask': (564,), 'num_templates': (), 'cluster_bias_mask': (512,), 'bert_mask': (512, 564), 'seq_mask': (564,), 'msa_mask': (512, 564)}
I0927 12:32:27.159153 47533421533248 model.py:176] Output shape was {'distogram': {'bin_edges': (63,), 'logits': (564, 564, 64)}, 'experimentally_resolved': {'logits': (564, 37)}, 'masked_msa': {'logits': (252, 564, 22)}, 'predicted_aligned_error': (564, 564), 'predicted_lddt': {'logits': (564, 50)}, 'structure_module': {'final_atom_mask': (564, 37), 'final_atom_positions': (564, 37, 3)}, 'plddt': (564,), 'aligned_confidence_probs': (564, 564, 64), 'max_predicted_aligned_error': (), 'ptm': (), 'iptm': (), 'ranking_confidence': ()}
I0927 12:32:27.160166 47533421533248 run_alphafold.py:212] Total JAX model model_4_multimer on alphafold predict time (includes compilation time, see --benchmark): 405.0s
I0927 12:32:36.790385 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:32:37.220777 47533421533248 amber_minimize.py:408] Minimizing protein, attempt 1 of 100.
I0927 12:32:37.785213 47533421533248 amber_minimize.py:69] Restraining 4388 / 8616 particles.
I0927 12:32:49.765658 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:32:51.099735 47533421533248 amber_minimize.py:500] Iteration completed: Einit 14217.70 Efinal -11929.10 Time 1.55 s num residue violations 0 num residue exclusions 0 
I0927 12:32:59.639316 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:33:00.460124 47533421533248 run_alphafold.py:198] Running model model_5_multimer on alphafold
I0927 12:33:00.460347 47533421533248 model.py:166] Running predict with shape(feat) = {'aatype': (564,), 'residue_index': (564,), 'seq_length': (), 'msa': (512, 564), 'num_alignments': (), 'template_aatype': (4, 564), 'template_all_atom_mask': (4, 564, 37), 'template_all_atom_positions': (4, 564, 37, 3), 'asym_id': (564,), 'sym_id': (564,), 'entity_id': (564,), 'deletion_matrix': (512, 564), 'deletion_mean': (564,), 'all_atom_mask': (564, 37), 'all_atom_positions': (564, 37, 3), 'assembly_num_chains': (), 'entity_mask': (564,), 'num_templates': (), 'cluster_bias_mask': (512,), 'bert_mask': (512, 564), 'seq_mask': (564,), 'msa_mask': (512, 564)}
I0927 12:39:44.934202 47533421533248 model.py:176] Output shape was {'distogram': {'bin_edges': (63,), 'logits': (564, 564, 64)}, 'experimentally_resolved': {'logits': (564, 37)}, 'masked_msa': {'logits': (252, 564, 22)}, 'predicted_aligned_error': (564, 564), 'predicted_lddt': {'logits': (564, 50)}, 'structure_module': {'final_atom_mask': (564, 37), 'final_atom_positions': (564, 37, 3)}, 'plddt': (564,), 'aligned_confidence_probs': (564, 564, 64), 'max_predicted_aligned_error': (), 'ptm': (), 'iptm': (), 'ranking_confidence': ()}
I0927 12:39:44.935096 47533421533248 run_alphafold.py:212] Total JAX model model_5_multimer on alphafold predict time (includes compilation time, see --benchmark): 404.5s
I0927 12:39:58.074591 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:40:02.555604 47533421533248 amber_minimize.py:408] Minimizing protein, attempt 1 of 100.
I0927 12:40:03.102931 47533421533248 amber_minimize.py:69] Restraining 4388 / 8616 particles.
I0927 12:40:11.388756 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:40:13.377688 47533421533248 amber_minimize.py:500] Iteration completed: Einit 41716.70 Efinal -12141.57 Time 1.50 s num residue violations 0 num residue exclusions 0 
I0927 12:40:23.150519 47533421533248 amber_minimize.py:177] alterations info: {'nonstandard_residues': [], 'removed_heterogens': set(), 'missing_residues': {}, 'missing_heavy_atoms': {}, 'missing_terminals': {<Residue 140 (HIS) of chain 0>: ['OXT'], <Residue 281 (HIS) of chain 1>: ['OXT'], <Residue 422 (HIS) of chain 2>: ['OXT'], <Residue 563 (HIS) of chain 3>: ['OXT']}, 'Se_in_MET': [], 'removed_chains': {0: []}}
I0927 12:40:23.987826 47533421533248 run_alphafold.py:279] Final timings for alphafold: {'features': 785.0464859008789, 'process_features_model_1_multimer': 1.239776611328125e-05, 'predict_and_compile_model_1_multimer': 482.38470673561096, 'relax_model_1_multimer': 48.56198525428772, 'process_features_model_2_multimer': 9.059906005859375e-06, 'predict_and_compile_model_2_multimer': 405.3013777732849, 'relax_model_2_multimer': 30.8317129611969, 'process_features_model_3_multimer': 1.6927719116210938e-05, 'predict_and_compile_model_3_multimer': 419.958025932312, 'relax_model_3_multimer': 32.85322308540344, 'process_features_model_4_multimer': 1.1205673217773438e-05, 'predict_and_compile_model_4_multimer': 404.9756181240082, 'relax_model_4_multimer': 32.027103900909424, 'process_features_model_5_multimer': 1.2159347534179688e-05, 'predict_and_compile_model_5_multimer': 404.4748296737671, 'relax_model_5_multimer': 37.697054386138916}
Traceback (most recent call last):
  File "/shared/ifbstor1/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/7fbec959cf2b/alphafold2/gen_extra_outputs.py", line 185, in <module>
    main()
  File "/shared/ifbstor1/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/7fbec959cf2b/alphafold2/gen_extra_outputs.py", line 180, in main
    residue_scores = generator.gen_residue_scores()
  File "/shared/ifbstor1/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/7fbec959cf2b/alphafold2/gen_extra_outputs.py", line 137, in gen_residue_scores
    for name, plddts in model_plddts.items()}
  File "/shared/ifbstor1/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/galaxy-australia/alphafold2/7fbec959cf2b/alphafold2/gen_extra_outputs.py", line 137, in <dictcomp>
    for name, plddts in model_plddts.items()}
KeyError: 'model_1'
```

**Comment by neoformit** [comment]
Looks like there's a part of the script that should have been made conditional on multimer.
@tchaussepiedifb thanks for letting us know, I'll look into it this week.

**Comment by neoformit** [comment]
Resolved with e1887a4579438640e48de8e3d46a31257c8d721a. This has been pushed to the toolshed - please request an update to version `2.1.2+galaxy4` to get the fix.
